docs/hiring-guide/index.rst

```diff
@@ -32,104 +32,35 @@ If you are willing to share your story with the community, you can contact jobs@
-2. What are poor questions that you may be tempted to ask as an interviewer, and how can you reframe these to be informative about a documentarian's skills.
-3. What can a candidate ask to learn about the interviewer's expectations of the role, how the successful candidate will be treated/valued, and the workplace culture (also around documentation).
-
-Some recommended interview questions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Self-starter:
-
-* How would you approach [a typical task for the role]?
-* Can you tell me about a time you initiated a project?
-* What would you do if you want to perform a project but don't get support from your teammates?
-
-Team player:
-
-* What do you imagine your ideal team being?
-* How would you go about teaching a new skill to your team?
-
-Experience in an organization:
-
-* If this were a new department/you were the first technical writer, where would you start?
-* Can you tell me about a time you showed leadership changing a process/procedure?
-* Do you think a writer should only write, or should they also edit existing texts/play additional roles?
-* How would you deal with the situation when senior staff members keep sending you (too many) emails to proofread/edit?
-* What's the thing you're most proud of in your professional life?
-* What's the hardest lesson you've had to learn at work?
-
-Dealing with feedback:
-
-* How would you deal with a senior member of your team who is giving you substandard text but you know they can do better?
-* How would you deal with feedback from a senior writer about your style when they are criticizing it but you believe you're right?
-* If a developer comes to you and they have to write a document, e.g. a readme, how would you get them started?
-* How do you keep up your concentration when you have a monotonous task?
-
-Personality:
-
-* How would your colleagues describe you (in one word)?
-
-Technical skills:
-
-* (Of the technologies named on their CV), can you tell me about your experience using this technology?
-* (Of a complex/feature-rich technology), can you tell me about some features you might use to manage a large volume of content/multiple different audiences/efficient updates...
-
-
-Interview questions better avoided (and alternatives)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-* Very in-depth questions about the subject matter, not about the writing skills.
-* Who are you as a person and what's your passion? -> Can you tell me about a project you worked on that you are proud of?
-* What's your five year plan? -> What's the next thing you want to learn?
-* What's your favorite food? (or other arbitrary ice-breakers) -> Anything more relevant to the job/workplace! Perhaps there's a social group in the office/common hobby to mention?
-* What would be your superpower? -> Would you rather be able to fly or be invisible? (to elicit their reasoning)
+Interview questions
```

Comment:
I'd add something like 

```
.. contents:: Table of Contents
    :depth: 3
```

around line 15 (and delete the tired note while you're there ;-)